scripts/exregional_run_met_genensprod_or_ensemblestat.sh

```diff
@@ -232,23 +231,28 @@ case "$OBTYPE" in
 esac
 vx_hr_end="${FCST_LEN_HRS}"
 
+set -x
```

Comment:
Just realized I accidentally left in several `set -x` debug lines; we probably want to remove those but I'll leave it up to you.